cvmfs/receiver/commit_processor.cc

```diff
@@ -138,19 +138,23 @@ CommitProcessor::Result CommitProcessor::Process(
   }
 
   const std::string public_key = "/etc/cvmfs/keys/" + repo_name + ".pub";
-  if (!server_tool->InitVerifyingSignatureManager(public_key)) {
+  const std::string certificate = "/etc/cvmfs/keys/" + repo_name + ".crt";
+  const std::string private_key = "/etc/cvmfs/keys/" + repo_name + ".key";
+  if (!server_tool->InitSignatureManager(public_key, certificate, private_key))
+  {
     LogCvmfs(
         kLogReceiver, kLogSyslogErr,
         "CommitProcessor - error: Could not initialize the signature manager");
     return kError;
   }
 
   shash::Any manifest_base_hash;
-  UniquePtr<manifest::Manifest> manifest(server_tool->FetchRemoteManifest(
+  const UniquePtr<manifest::Manifest> manifest_tgt(
```

Comment:
what does the `tgt` in `manifest_tgt` stand for?

Reply:
I'm assuming "target"